app/controllers/concerns/taxonomy_navigation.rb

```diff
@@ -1,18 +1,12 @@
 module TaxonomyNavigation
   def should_present_taxonomy_navigation?(content_item)
-    content_item.present? && (tagged_to_world_wide_taxonomy?(content_item) ||
```

Comment:
@tijmenb we've employed similar logic in smart-answers, are they going to suffer from a similar problem?

Reply:
Yes, we should remove the special casing there too. I've added it to  https://trello.com/c/TqnWwFif/101-mop-up-issues-with-the-world-wide-taxonomy.